systori/apps/project/api.py

```diff
@@ -63,6 +96,11 @@ def exists(self, request, pk=None):
                 {"name": _("Project not found.")}, status=HTTP_206_PARTIAL_CONTENT
             )  # name is expected by the JS Client
 
+    project_search_response = openapi.Response(
+        "List of projects found", ProjectSearchResultSerializer
```

Comment:
it's only a single match, if any - so not a List of projects found but only a single project found

Reply:
The `search` method actually returns a list of projects (based on the line 142 and the response has `projects` key). This is to tell the flutter client that the response is going to be 
```json
{
  "projects": [
   // project objects
   ]
}
```

Even though it is a single match, it still is a list with a single element.

Existing code understands this and is working so I don't want to change the response.